### ISSUE TITLE
VMNodeScrape selector not works

### ISSUE THREAD

**Issue by CRASH-Tech** [VictoriaMetrics/operator#644]
I'm trying to scape metrics only from control-plane nodes, but vmagent scrapes from all nodes.

I was trying this, but it's not work:
```
  selector:
    matchExpressions:
      - key: "node-role.kubernetes.io/control-plane"
        operator: Exists
```
```
  selector:
    matchLabels:
      node-role.kubernetes.io/control-plane: ""
```

**Comment by f41gh7** [comment]
Hello, could you add a full example of VMNodeScrape object?

**Comment by CRASH-Tech** [comment]
> Hello, could you add a full example of VMNodeScrape object?

```
apiVersion: operator.victoriametrics.com/v1beta1
kind: VMNodeScrape
metadata:
  name: kubernetes-etcd
spec:
  path: /metrics
  port: "2381"
  scrape_interval: 15s
  selector:
    matchExpressions:
      - key: "node-role.kubernetes.io/control-plane"
        operator: Exists
```

**Comment by CRASH-Tech** [comment]
Some news
```
  selector:
    matchExpressions:
      - key: node-role.kubernetes.io/control-plane
        operator: Exists
```
Do not scrape at all

```
  selector:
    matchLabels:
      node-role.kubernetes.io/control-plane: ""
```
Scrape from all nodes

**Comment by Amper** [comment]
Duplicate of #645

**Comment by f41gh7** [comment]
An issue must be fixed [v0.34.0](https://github.com/VictoriaMetrics/operator/releases/tag/v0.34.0) release
